### ISSUE TITLE
Move migration code into Debian scripts

### ISSUE THREAD

**Issue by jdknives** [skycoin/skywire#1070]
**Feature description**

We currently handle config migrations and other upgrading procedures in the Skywire Visor binary as we previously did not have a pacakging system available to do these checks with. That is less than optimal and should be moved out of the binary and into the Debian scripts as it is easier to handle and should be less error prone hopefully.

**Comment by 0pcom** [comment]
This is already present in the packages if I understand you correctly.

After the first time I ran into issues with the config not working with the upgraded visor, I started generating (or regenerating) the config with skywire-cli in the skywire-autoconfig script on every update of the package. That script is called on installation and updates as part of the postinstall process.

The config for the visor mode (remote hypervisor) is not currently updated because it is created by the user

but the only reason this config was not updated was, at the time, the flag to retain the hypervisor configuration was not present.

~~We have that now and~~ it's an easy check for the presence of the second config file and to update it if it exists, with the flag to retain the configured hypervisors (`-x` flag?)

@mrpalide I cannot find that flag anywhere, I thought you had told me about it previously?

On the next versioned release of skywire all the fixes will be integrated to the skywire-autoconfig script in the packages.

It should be safe to remove the config migration / update code from the visor for the next release.
